stock_picking_tier_validation/models/stock_picking.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class Picking(models.Model):
+    _name = "stock.picking"
+    _inherit = ["stock.picking", "tier.validation"]
+    _state_from = ["draft", "waiting", "confirmed", "assigned"]
```

Comment:
```suggestion
    _state_from = ["waiting", "confirmed", "assigned"]
```
Validate button doesn't show when the state is draft.